### ISSUE TITLE
[A11Y] Signals Link is not Visually Evident

### ISSUE THREAD

**Issue by lee-cooper** [JeffryGonzalez/applied-angular#2]
## Describe the bug

The signals link does not have a visual indicator to distinguish it from other elements on the page (such as paragraphs) and signal (hah, get it?) that it is an actionable.

See [WCAG 1.4.1 (Use of Color)](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) and [this additional resource](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73) for understanding the WCAG failure, including examples.

## Screenshot(s)
<img width="532" alt="image" src="https://github.com/user-attachments/assets/31e9037d-eada-4ac0-820a-f16e3c67c90c">

## Recommendation
The simplest fix would be to add an underline to the link to help distinguish it from the other elements on the page. Additionally, adding a color to the link **in addition** to the underline would further enhance this, providing multiple indicators to users that this can be interacted with.

## Browser(s)
Chrome
